styles/styles.css

```diff
@@ -238,8 +239,8 @@ h6 {
 }
 
 p, dl, ol, ul, pre, blockquote {
-    margin-top: 1em;
-    margin-bottom: 1em;
+    margin-top: 12px;
```

Comment:
this is a bit too generic, I think it may be better to update them only for the accordion-1 block
Found one issue:
![image](https://github.com/hlxsites/bevespi/assets/26941572/7467e6c7-3743-4eff-a034-1ae343b0e8df)
the top margin of ul should be 0

Reply:
> this is a bit too generic, I think it may be better to update them only for the accordion-1 block Found one issue: ![image](https://user-images.githubusercontent.com/26941572/278597317-7467e6c7-3743-4eff-a034-1ae343b0e8df.png) the top margin of ul should be 0

@lansami ul margin 0 fixed.

regarding general margin: I found myself correcting it everywhere; i think in some cases we just need to change the value from the boilerplate. Nobody is telling us that 1em is right for bevespi, it was just the boilerplate value. WDYT?

Reply:
I agree, I think we can use the value that suits better most of the cases and override where needed